tests/release-notes-test.ts

```diff
@@ -0,0 +1,68 @@
+jest.mock("danger", () => jest.fn());
+import danger from "danger";
+const dm = danger as any;
+
+import checkReleaseNotes from "../org/pr/release-notes";
+
+// The mocked data and return values for calls the rule makes.
+beforeEach(() => {
+    dm.warn = jest.fn().mockReturnValue(true);
+
+    dm.danger = {
+        git: {
+            modified_files: ["RELEASE-NOTES.txt"],
+            created_files: [],
+            deleted_files: []
+        },
+        github: {
+            pr: {
+                base: {
+                    ref: "release/1.0",
+                }
+            },
+            issue: {
+                labels: []
+            },
+        },
+    };
+});
+
+describe("release notes checks", () => {
+    it("warns when RELEASE-NOTES.txt is changed in a PR which targets a release branch", async () => {
+        await checkReleaseNotes();
+        
+        expect(dm.warn).toHaveBeenCalledWith(expect.stringContaining("This PR contains changes to RELEASE_NOTES.txt.\n"));
```

Comment:
nit to only apply if you applied the previous one too
```suggestion
        expect(dm.warn).toHaveBeenCalledWith(expect.stringContaining("This PR contains changes to \`RELEASE_NOTES.txt\`.\n"));
```